src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -117,15 +132,27 @@ const CreateNewListModal: React.FC<CreateNewListModalProps> = ({
             onClose={onClose}
             title={t("collectorSaves.createNewListModal.title")}
             data-testid="CreateNewList"
+            header={
+              artwork ? <CreateNewListModalHeader artwork={artwork} /> : null
+            }
             footer={
               <>
                 {/* Desktop view */}
                 <Media greaterThanOrEqual="sm">
-                  <Flex justifyContent="flex-end">
+                  <Flex
+                    justifyContent={onBackClick ? "space-between" : "flex-end"}
+                  >
+                    {!!onBackClick && (
+                      <Button variant="secondaryBlack" onClick={onBackClick}>
+                        {t("collectorSaves.createNewListModal.back")}
+                      </Button>
+                    )}
```

Comment:
Thanks for adding this button here ([Ryan confirmed](https://artsy.slack.com/archives/C9SATFLUU/p1678099239933699?thread_ts=1677803388.476209&cid=C9SATFLUU) that we should always display the back/cancel button in both desktop and mweb contexts)

Do we also need to modify the behavior of the mweb version below on L175?

Reply:
Addressed here: https://github.com/artsy/force/pull/12021/commits
Thank you for clarifications!